.gitlab-ci.yml

```diff
@@ -182,9 +187,9 @@ pup5_latest-unit:
   <<: *cache_bundler
   <<: *setup_bundler_env
   <<: *spec_tests
-  allow_failure: true
-
 
+# Acceptance Tests
```

Comment:
We have been adding an acceptance stage for 5.5 to our other tests, we should probably do the same here.